jni-gen/src/wrapper_generator.rs

```diff
@@ -62,6 +66,9 @@ impl WrapperGenerator {
 
         //remove_dir_all(out_dir)?;
         create_dir_all(out_dir)?;
+        create_dir_all(out_dir.join("builtins"))?;
+        let builtins_utils = Path::new("/home/simon/repo/prusti-dev/jni-gen/builtins/utils.rs");
+        copy(builtins_utils, out_dir.join("builtins/utils.rs"))?;
```

Comment:
Please use the `walkdir` crate to recursively copy the content of the folder
```rust
for entry in walkdir::WalkDir::new(builtins_dir) { /* do the copy */ }
```

Reply:
Not applicable anymore as changed the code ([commit](https://github.com/viperproject/prusti-dev/pull/1364/commits/faade4e4854e610090de7fd487a5d15cacab2451)) according to Aurel's [suggestion](https://github.com/viperproject/prusti-dev/pull/1364#discussion_r1135616848).